snippets/product-card-placeholder.liquid

```diff
@@ -8,17 +8,17 @@
 <div class="card-wrapper">
   <a class="full-width-link"><span class="visually-hidden">{{ 'onboarding.product_title' | t }}</span></a>
 
-  <div class="card card--text-only card--soft card--product">
-    <div class="card__inner">
-      <div class="card__content"><h2 class="card__text h3">{{ 'onboarding.product_title' | t }}</h2></div>
-    </div>
-  </div>
-
   <div class="card-information">
     <div class="card-information__wrapper">
       <span class="visually-hidden">{{ 'accessibility.vendor' | t }}</span>
       <div class="caption-with-letter-spacing light">{{ 'products.product.vendor' | t }}</div>
       {% render 'price' %}
     </div>
   </div>
+
+  <div class="card card--text-only card--soft card--product">
+    <div class="card__inner">
+      <div class="card__content"><h2 class="card__text h3">{{ 'onboarding.product_title' | t }}</h2></div>
+    </div>
+  </div>  
 </div>
```

Comment:
Small nitpick but on the product page, when scaling the body size, the spacing between some elements feel a bit tight: https://screenshot.click/05-17-vobf4-raq5b.mp4


cc @melissaperreault

Reply:
@KaichenWang Is it possible that in this particular block space that the spacing is not scaled?